src/LoginForm.jsx

```diff
@@ -151,13 +151,7 @@ class LoginForm extends React.Component {
     this.setState({test: true})
     this._testStep('biz', 'pending');
     let client = this.props.createClient();
-    let testUpdateLoop = null;
 
-
-    window.onunload = () => {
-      cleanup()
```

Comment:
`cleanup()` does not exist anywhere. I assume this should be `this._cleanup()`, but when I was trying to call it here, it did not execute at all. Maybe `onunload` is not allowed to execute async code?